File: web3rpc/rpc-specs/code-samples/python/admin/setMaxSubscriptionPerWSConn.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

limit = 5

w3 = Web3(Web3.HTTPProvider(host))
admin_response = w3.geth.admin.set_max_subscription_per_ws_conn(limit)

print(admin_response)

File: web3rpc/rpc-specs/code-samples/javascript/admin/setMaxSubscriptionPerWSConn.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const maxSubscriptionPerWSConn = 5;

  provider.admin
    .setMaxSubscriptionPerWSConn(
      maxSubscriptionPerWSConn,
      {},
      (err, data, response) => {},
    )
    .then((data) => {
      console.log(data);
    });
})();
